### PR TITLE
created PlaceTable

### PR DESCRIPTION
**Pull request by NazarMykhailiv**

### CODE REVIEW

File: src/components/PlacesTable/PlacesTable.scss [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  }
+}
+
+.background__heart,
+.background__time,
+.background__type {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (TaniaMitnovych) [inline src/components/PlacesTable/PlacesTable.scss:27]:
```
@include flex($content:center, $items:center);
```

File: src/components/PlacesTable/PlacesTable.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+.place__description {
+    padding: 0px 10px 0px 5px;
+    margin-top: 5px;
+    overflow: hidden;
+    word-wrap: break-word;
+    text-overflow: ellipsis;
+}
+
+@media screen and (max-width: 1024px){
```
Comment (TaniaMitnovych) [inline src/components/PlacesTable/PlacesTable.scss:71]: use media helpers from mixins.scss

File: src/components/PlacesTable/PlacesTable.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  .place__container{
+    margin: auto 30px 30px ;
+  }
+}  
+
+@media screen and (max-width: 768px) {
```
Comment (TaniaMitnovych) [inline src/components/PlacesTable/PlacesTable.scss:77]: use media helpers from mixins.scss

File: src/components/PlacesTable/PlacesTable.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    };
+    
+    window.addEventListener("resize", handleResize);
+
+    handleResize();
+
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
```
Comment (TaniaMitnovych) [inline src/components/PlacesTable/PlacesTable.tsx:46]: its better to use css grids for these purposes

File: src/components/PlacesTable/PlacesTable.tsx [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+            ))}
+          </div>
+          <p className="place__name">{item.name}</p>
+          <p className="place__description">{item.description}</p>
+        </div>
+      </td>
+    ));
+
+    tableRows.push(<tr key={i}>{tableCells}</tr>);
+  }
```
Comment (TaniaMitnovych) [inline src/components/PlacesTable/PlacesTable.tsx:85]: You should create separate component and use it as a place card passing needed info as a props

File: src/assets/icons.tsx [hunk 6]
```diff
@@ -61,7 +61,7 @@ export const CloseIcon: React.FC<IconProps> = ({width, height, fill}) => {
 
 export const LocationIcon = () => {
     return (
-        <svg xmlns="http://www.w3.org/2000/svg" fill='#e27976' viewBox="0 0 384 512">
+        <svg xmlns="http://www.w3.org/2000/svg" fill='#e27976' viewBox="0 0 384 512" className="LocIcon">
```
Comment (OlehYavoriv) [inline src/assets/icons.tsx:64]: don't use className, use IconProps to customize the icon

File: src/components/PlaceCard/PlaceCard.scss [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+    text-overflow: ellipsis; 
+    white-space: nowrap; 
+}
+  
+.place__description {
+      padding: 0px 10px 0px 5px;
+      margin-top: 5px;
+      overflow: hidden;
+      word-wrap: break-word;
+      text-overflow: ellipsis;
```
Comment (OlehYavoriv) [inline src/components/PlaceCard/PlaceCard.scss:111]: use mixin (dots)

**Review by OlehYavoriv — APPROVED** [review]

**Review by TaniaMitnovych — APPROVED** [review]

**Pull request merged**